client/src/main/java/model/Fleet.java

```diff
@@ -27,7 +27,7 @@ public void addShip(Ship ship) {
         .collect(Collectors.toList());
   }
 
-  public boolean isFleetEmpty() {
-    return ships.isEmpty();
+  public boolean isFleetFullyBuilt() {
+    return ships.size() == 10;
```

Comment:
magic number :D Probably 10 is a size of a board. Don't you want to keep it as constant?